Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: test lint autoformat run ftest install release docker-build docker-run docker-push
 
-PYTHON=python3.10
```

Comment:
this change requires your setup to have an alias for python, in my machine I didn't had that and to make it work I have to do `PYTHON=python3 make install`

Is this an issue ? If no do we want to mention it in somewhere?

Reply:
Opened discussion in https://elastic.slack.com/archives/C01795T48LQ/p1723648672552739?thread_ts=1723465470.280719&cid=C01795T48LQ